File: pset1/water.rb
def bottles_used
  shower_minutes = get_minutes
  gallons_used = get_gallons(shower_minutes)
  ounces_used = get_ounces(gallons_used)
  bottles_used = get_bottles(ounces_used)
  puts "You used #{bottles_used} bottles of water."
end

def get_minutes
  puts 'Enter length in minutes of you shower: '
  shower_length = gets.chomp.to_i
end

def get_gallons(shower_minutes)
  shower_minutes * 1.5
end

def get_ounces(gallons_used)
  gallons_used * 128
end

def get_bottles(ounces_used)
  ounces_used / 16
end

bottles_used

File: pset1/water.js
let readline = require('readline');

let rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

function getBottles() {
  rl.question('Enter the length of your shower in minutes: ', function(lengthOfShower) {
    let gallonsUsed = getGallons(lengthOfShower);
    let ouncesUsed = getOunces(gallonsUsed);
    console.log(`You used ${ouncesUsed / 16} bottles of water!`);
    rl.close();
  })
}

function getGallons(lengthOfShower) {
  return lengthOfShower * 1.5;
}

function getOunces(gallonsUsed) {
  return gallonsUsed * 128;
}

getBottles();
